### ISSUE TITLE
Advanced sub-schematics won't spawn other advanced sub-schematics sometimes

### ISSUE THREAD

**Issue by KingofAnomaly** [ryandw11/CustomStructures#148]
**Describe the bug**
Advanced sub-schematics don't attempt to spawn in other advanced sub-schematics sometimes leaving the signs behind.
I have gotten it to work on smaller structures so I'm not sure what is causing it.

**Steps to Reproduce**
Have a large advanced sub-schematic with other advanced sub-schematic signs.

**Expected behavior**
After the sub-schematic spawns it spawns the next set of sub-schematics 

**Screenshots**
![Screen Shot 2022-07-11 at 11 07 40 AM](https://user-images.githubusercontent.com/48358224/178296307-e0ec5d1d-92b7-4abe-8eb5-28a6103f99a8.png)

![Screen Shot 2022-07-11 at 10 42 41 AM](https://user-images.githubusercontent.com/48358224/178290946-126f300d-8379-4aa1-8306-d35fde929655.png)

**Structure File(s)**
https://pastebin.com/e1TBYqKp

**Environment Information:**
 - Minecraft Version: 1.18.2
 - Spigot Version: 1.18.2-320
 - Custom Structures Version: 1.8.1
 - World Edit Version: 7.2.10
 - Async World Edit Version: 3.9.4

**Additional context**
I am trying to make a dungeon with randomized walls and decorations. There is a main chamber which is the structure schematic and I have sub-schematics for paths expanding out from the center, sub-schematics for different wall variations and wall facades. The paths should be placed then the walls then the facade.

**Comment by IceWolf23** [comment]
Any news about this ? @ryandw11

**Comment by KingofAnomaly** [comment]
It's been over a year since their last update I think the plugin is not being actively developed anymore and I've just decided to do it the vanilla way with jigsaw blocks.

**Comment by IceWolf23** [comment]
Yeah, I suppose, but you know, trying asking doesn't hurt anyone xd, anyway if you still need it I am trying to update the plugin to 1.21.10 x PaperMC

**Comment by KingofAnomaly** [comment]
What's your servers name? I'm curious to see what kind of server you are running.

**Comment by IceWolf23** [comment]
> What's your servers name?

CrystalSMP, not yet online btw

**Comment by KingofAnomaly** [comment]
Oh you just have it for friends or something? I just wanted to see some of your work because I was curious whether or not to ask you if you wanted to help out on my server. I kind of have too many projects and not enough time to work on them all.

**Comment by IceWolf23** [comment]
> Oh you just have it for friends or something? I just wanted to see some of your work because I was curious whether or not to ask you if you wanted to help out on my server. I kind of have too many projects and not enough time to work on them all.

I'm trying to bring back a Minecraft Server's Community where I was Admin some years ago, but the server closed and the whole community kind of shades off

**Comment by KingofAnomaly** [comment]
Oh I know what you mean I have had that happen a few time which is why I started running my server. What kind of server was it?

**Comment by IceWolf23** [comment]
> Oh I know what you mean I have had that happen a few time which is why I started running my server. What kind of server was it?

A little network, 
Minigames, arcades, prison,
We had a custom survival in 1.16.5, with custom items, dungeons, cool stuff etc, the most awaited event on our server was a live-music giveaway in game, we managed to have litteraly a live DJ to play songs during the giveaway, crazy stuff for real xd

Our server had near 100 on allways
